scripts/ansible/create-policy-and-role.retry

```diff
@@ -0,0 +1 @@
+localhost
```

Comment:
I think this retry file is not related to these changes

Reply:
Maybe retry files should be added to the gitignore since we are introducing ansible playbooks to the repo?